docs/admin/templates/extending-templates/parameters.md

```diff
@@ -391,3 +391,546 @@ parameters in one of two ways:
   ```
 
   Or set the [environment variable](../../setup/index.md), `CODER_EXPERIMENTS=auto-fill-parameters`
+
+## Dynamic Parameters
+
+Dynamic Parameters enhances Coder's existing parameter system with real-time validation,
+conditional parameter behavior, and richer input types.
+This feature allows template authors to create more interactive and responsive workspace creation experiences.
+
+### Enable Dynamic Parameters
+
+<div class="tabs">
+
+To use Dynamic Parameters, enable the experiment flag or set the environment variable:
+
+#### Flag
+
+```shell
+coder server --experiments=dynamic-parameters
+```
+
+#### Env Variable
+
+```shell
+CODER_EXPERIMENTS=dynamic-parameters
+```
+
+</div>
+
+Once enabled, users can toggle between the experimental and classic interfaces during
+workspace creation.
+
+## Features and Capabilities
+
+Dynamic Parameters introduces three primary enhancements to the standard parameter system:
+
+- **Conditional Parameters**
+
+  - Parameters can respond to changes in other parameters
+  - Show or hide parameters based on other selections
+  - Modify validation rules conditionally
+  - Create branching paths in workspace creation forms
+
+- **Dynamic Data Fetching**
+
+  - Fetch data at workspace creation time
+  - Access Coder user attributes (groups, username, auth)
+  - Selectively expose options based on user properties
+  - Connect to external resources
+
+- **Rich Parameter Entry**
```

Comment:
We are trying to scrap the word `Rich`. We released `Rich Parameters`, and we want to bring that name back to just `Parameters` now.

```suggestion
- **Additional Form Inputs**
```